From acbd8b0c40e0421699048e40f3d59ff01287b7b6 Mon Sep 17 00:00:00 2001
From: William Light <git@wrl.lhiaudio.com>
Date: Thu, 25 Feb 2021 14:04:02 +0000
Subject: [PATCH] js/src/build/js.pc.in: Fix cross-compiled pkg-config
 definitions

-isystem flags have PKG_CONFIG_SYSROOT_DIR prepended to them
automagically, -include flags don't, so we have to add that manually.

Signed-off-by: William Light <git@wrl.lhiaudio.com>
---
 js/src/build/js.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/src/build/js.pc.in b/js/src/build/js.pc.in
index d40deb1..ec1332e 100644
--- a/js/src/build/js.pc.in
+++ b/js/src/build/js.pc.in
@@ -11,4 +11,4 @@ Libs: -L${libdir} -l@JS_LIBRARY_NAME@
 # Use -isystem for includes rather than -I, to mark them as "system headers"
 # that don't generate warning diagnostics. For justification, see
 # https://bugzilla.mozilla.org/show_bug.cgi?id=1539036
-Cflags: -include ${includedir}/@JS_LIBRARY_NAME@/js/RequiredDefines.h -isystem ${includedir}/@JS_LIBRARY_NAME@
+Cflags: -include ${pc_sysrootdir}${includedir}/@JS_LIBRARY_NAME@/js/RequiredDefines.h -isystem ${includedir}/@JS_LIBRARY_NAME@
-- 
2.26.2

